lib/noop_tracer.dart

```diff
@@ -26,6 +26,12 @@
 
 library opentracing_noop;
 
+export 'src/noop_scope.dart';
+export 'src/noop_scope_manager.dart';
 export 'src/noop_span.dart';
 export 'src/noop_span_context.dart';
 export 'src/noop_tracer.dart';
+export 'src/reference.dart';
+export 'src/reference.dart';
+export 'src/span_context.dart';
```

Comment:
Also, do we need to export this stuff through Noop Tracer? It makes more sense to me that they would import the normal library entrypoint to get access to these things, and use this entrypoint for just the Noop-related exports.